src/javascript/app/base/redirect_popup.js

```diff
@@ -0,0 +1,55 @@
+const BinarySocket     = require('./socket');
+const isEuCountry      = require('../common/country_base').isEuCountry;
+const Client           = require('../base/client');
+const Login            = require('../../_common/base/login');
```

Comment:
too much whitespace before the `=`

Reply:
Yes, I don’t like it either. I'm just trying to do same as most of the codes I've seen in binary-static.
I'm removing them.